linode/manager
# refactor: [M3-6417] - SRC > Components > Tile

## Description 📝

- Migrate Tile component from JSS to tss-react
- Use named exports instead of default

## How to test 🧪

- Check places in the app where Tile are used
- Check that the Tile story renders in Storybook
